pynotify/handlers.py

```diff
@@ -120,7 +126,7 @@ def _can_handle(self):
         """
         Returns ``True`` if handler should handle creating of notification(s).
         """
-        return True
+        return not self._admin_template or self._admin_template and self._admin_template.is_active
```

Comment:
there can be

`not self._admin_template or self._admin_template.is_active`

the middle part is not required